### PR TITLE
add delete job

### PR DESCRIPTION
**Pull request by coryatwater-FW**

### CODE REVIEW

File: apps/api/src/migration/1660225963147-AddDeleteCascadeToJobStatusAudit.ts
Comment (sunghwan-park-fw) [inline apps/api/src/migration/1660225963147-AddDeleteCascadeToJobStatusAudit.ts:15]: Is this right that `up` and `down` are same?

File: apps/web/src/services/applicant.ts
Comment (sunghwan-park-fw) [inline apps/web/src/services/applicant.ts:119]: You don't need to call notifyError here. All axios errors are notified by axios interceptor

File: apps/web/src/components/milestone-logs/recruitment/Record.tsx
Comment (sunghwan-park-fw) [inline apps/web/src/components/milestone-logs/recruitment/Record.tsx:118]: It might a rare case but what if users with applicant write access added a job and then they lose the access later? can they still see the delete button?

**Review by sunghwan-park-fw — APPROVED** [review]
See my comments

File: apps/api/src/migration/1660225963147-AddDeleteCascadeToJobStatusAudit.ts
Comment (coryatwater-FW) [reply apps/api/src/migration/1660225963147-AddDeleteCascadeToJobStatusAudit.ts:15]: up contains 'ON DELETE CASCADE', down contains 'ON DELETE NO ACTION'

File: apps/web/src/components/milestone-logs/recruitment/Record.tsx
Comment (coryatwater-FW) [reply apps/web/src/components/milestone-logs/recruitment/Record.tsx:118]: no, it'd be blocked by the role mask

**Pull request merged**
